### PR TITLE
report style changes

### PR DESCRIPTION
**Pull request by binaryberserker**
TR-142: Copy button UI
TR-143: Save report and Reset Changes UI

### CODE REVIEW

**Comment by binaryberserker** [comment]
https://github.com/user-attachments/assets/7e7d5352-59d7-4b9a-94eb-2d3f29bbcc7d

File: public/react/SaveReports/SaveReports.tsx [hunk 1]
```diff
@@ -2,14 +2,13 @@ import type { FC } from "react";
...
 import React, { useEffect, useState } from "react";
-import DropdownMenu, { DropdownItem } from "@atlaskit/dropdown-menu";
 import { v4 as uuidv4 } from "uuid";
 import Button from "@atlaskit/button/new";
-import ChevronDown from "@atlaskit/icon/glyph/chevron-down";
 
 import { useAllReports, useCreateReport, useRecentReports } from "../services/reports";
 import SaveReportModal from "./components/SaveReportModal";
 import SavedReportDropdown from "./components/SavedReportDropdown";
+import ReportControls from "./components/SavedReportDropdown/ReportControls";
```
Comment (DavidNic11) [inline public/react/SaveReports/SaveReports.tsx:11]: Why is `ReportCntrols` going inside the SavedReportDropdown Modlet?

File: public/react/SaveReports/components/SavedReportDropdown/ReportControls.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  isDirty?: boolean;
+  updateSelectedReport?: () => void;
+  openModal?: () => void;
+  resetChanges?: () => void;
+}
+
+const ReportControls: FC<ReportControlProps> = ({
+  hasSelectedReport = false,
+  isDirty = false,
+  updateSelectedReport = () => {},
```
Comment (DavidNic11) [inline public/react/SaveReports/components/SavedReportDropdown/ReportControls.tsx:20]: optional functions don't need to be defaulted

Comment (binaryberserker) [reply public/react/SaveReports/components/SavedReportDropdown/ReportControls.tsx:20]: if they dont have a default then you have to do an additional check before calling them

File: public/react/SaveReports/SaveReports.tsx (hunk 1 above)
Comment (binaryberserker) [reply public/react/SaveReports/SaveReports.tsx]: fixed

**Review by DavidNic11 — APPROVED** [review]

**Pull request merged**